File: Spipama.Application/ViewModels/NewsViewModel.cs
using SPIPAMA.Domain.BaseModel;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Spipama.Application.ViewModels
{
   public class NewsViewModel : BaseModel
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public string ImageUrl { get; set; }
        public DateTime DateEvent { get; set; }
    }
}

File: Spipama.API/SpipamaClientPublicWebApp/src/app/@core/viewModels/NewsViewModel.ts
export class NewsViewModel {
  title: string;
  description: string;
  imageUrl: string;
  dateEvent: Date;
  createdOn: Date;

  constructor(
    title: string = null,
    description: string = null,
    imageUrl: string = null,
    dateEvent: Date = null,
    createdOn: Date = null,
  ) {
      this.title = title;
      this.description = description;
      this.imageUrl = imageUrl;
      this.dateEvent = dateEvent;
      this.createdOn = createdOn;
  }
}
